Bottom: 2af6889372ea0b4c177eeee226bca43f0aaa4674
Top:    f92d65dd29a35f83009830cafaa5fb7f151da3e2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 16:09:08 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
